index.html

```diff
@@ -0,0 +1,156 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>portfolio</title>
+</head>
+<body>
+  <div class="content">
+  <header class="site-header">
+    <div class="site-wrapper">header</div>
+  </header>
+  <main class="site-main">
+    <section class="first-section">
```

Comment:
Each section must have heading. Please see this links: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section